packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -284,59 +304,51 @@ function FinancesApp(props) {
 
   return (
     <Router history={patchedHistory}>
-      <CompatRouter>
```

Comment:
Remove the compat router -- the rest is indentation here until the <Route> block.